file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java

```diff
@@ -33,11 +33,14 @@
 import org.apache.zeppelin.interpreter.InterpreterContext;
 import org.apache.zeppelin.interpreter.InterpreterException;
 import org.apache.zeppelin.interpreter.thrift.InterpreterCompletion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * HDFS implementation of File interpreter for Zeppelin.
  */
 public class HDFSFileInterpreter extends FileInterpreter {
+  private static final Logger LOGGER = LoggerFactory.getLogger(HDFSFileInterpreter.class);
```

Comment:
:+1: 
It is good that these changes are included with this unification.

Reply:
:)